packages/chart/src/components/LineChart.tsx

```diff
@@ -36,9 +36,63 @@ export default function LineChart({ xScale, yScale, getXAxisData, getYAxisData,
     return `${formatNumber(value, axis)}`
   }
 
+  const isNumber = value => {
+    // in debugging I saw cases where inbound was a 'number'
+    // and other times a 'string' so might as well take care of both here
+    if (typeof value === 'number') {
+      return !Number.isNaN(value)
+    }
+    if (typeof value === 'string') {
+      return value !== null && value !== '' && /\d+\.?\d*/.test(value)
+    }
+    return false // if we get here something is wrong so return false
+  }
+
+  const checkCy = cy => {
```

Comment:
That was mainly for initial debugging bc I thought the cy input was causing it to crash.  It could possibly be removed.  If it's needed I can revisit the name for sure.

Reply:
Confirmed it's not needed and removed it.